content/docs/csidriver/installation/helm/powerflex.md

```diff
@@ -151,7 +151,7 @@ Use the below command to replace or update the secret:
 
 | Parameter                | Description                                                                                                                                                                                                                                                                                                                                                                                                    | Required | Default |
 | ------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- | ------- |
-| version | Set to verify the values file version matches driver version and used to pull the image as part of the image name. | Yes | 2.7.0 |
+| version | Set to verify the values file version matches driver version and used to pull the image as part of the image name. | Yes | 2.7.1 |
```

Comment:
Set to verify that the file version matches the driver version. 
I don't understand the next part. I think this should be a separate sentence. Is the file/driver version part of the image name? Can you please clarify.

Reply:
@sharont58 
Yes the driver version is part of the image name.

Reply:
How about: 
"Set to verify that the file version matches the driver version. When you pull the image, the driver version is part of the image name."

Reply:
Actually what that refers is https://github.com/dell/csi-powerflex/blob/main/helm/csi-vxflexos/values.yaml#L6 this is the values file. If we mention the version here as v2.7.1 this matches the driver version and then it is used to pull the image i.e dellemc/csi-vxflexos:v2.7.1 as part of the image name.

Reply:
@sharont58  let me know if it is still causing confusion.

Reply:
I see. Thanks for the info. I'm good with what you have there.

Reply:
Thanks